### PR TITLE
Basic search

### PR DESCRIPTION
**Pull request by EKustova**

### CODE REVIEW

File: ChainReactionBindings/Bindings/SearchPageBindings.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+namespace ChainReactionBindings.Bindings
+{
+	[Binding]
+	class SearchPageBindings
+	{
+		SearchPage page = new SearchPage();
+
+		[Then(@"I see the search page is loaded")]
+		public void SearchPageLoaded()
+		{
```
Comment (etolmachev) [inline ChainReactionBindings/Bindings/SearchPageBindings.cs:16]: Add Browser.WaitReadyState()

File: ChainReactionBindings/Bindings/SearchPageBindings.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+			if (!page.SearchPageLoaded())
+			{
+				throw new Exception("Search page is not loaded!");
+			}
+		}
+
+		[Then(@"I see the empty search page is loaded")]
+		public void EmptySearchPageLoaded()
+		{
```
Comment (etolmachev) [inline ChainReactionBindings/Bindings/SearchPageBindings.cs:25]: Add Browser.WaitReadyState()

File: ChainReactionBindings/Bindings/SearchPageBindings.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+			if (!page.EmptySearchPageLoaded())
+			{
+				throw new Exception("Empty search page is not loaded!");
+			}
+		}
+
+		[Then(@"I see ""(.*)"" message on search page")]
+		public void ThenISeeMessageOnSearchPage(string text)
+		{
+			switch (text)
```
Comment (etolmachev) [inline ChainReactionBindings/Bindings/SearchPageBindings.cs]: Switch is not an option here, just use parameters and form them to expected message, then compare with actual message

File: ChainReactionBindings/Bindings/SearchPageBindings.cs [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+				default:
+					throw new Exception(string.Format("Field {0} is not implemented", text));
+			}
+			
+		}
+
+		[Then(@"I see ""(.*)"" items on search page")]
+		public void ThenISeeItemsOnSearchPage(string count)
+		{
+			Assert.AreEqual(count, page.ItemsCountElement.FindElements(By.ClassName("products_details_container")).Count.ToString());
```
Comment (etolmachev) [inline ChainReactionBindings/Bindings/SearchPageBindings.cs]: Compare in int, also pass count parameter as int to the bindings

**Review by etolmachev — CHANGES_REQUESTED** [review]

File: ChainReactionBindings/Bindings/SearchPageBindings.cs (hunk 1 above)
Comment (EKustova) [reply ChainReactionBindings/Bindings/SearchPageBindings.cs:16]: discused

File: ChainReactionBindings/Bindings/SearchPageBindings.cs (hunk 4 above)
Comment (EKustova) [reply ChainReactionBindings/Bindings/SearchPageBindings.cs]: done

File: ChainReactionBindings/Bindings/SearchPageBindings.cs (hunk 2 above)
Comment (EKustova) [reply ChainReactionBindings/Bindings/SearchPageBindings.cs:25]: discused

File: ChainReactionBindings/Bindings/SearchPageBindings.cs (hunk 3 above)
Comment (EKustova) [reply ChainReactionBindings/Bindings/SearchPageBindings.cs]: done

**Pull request merged**